cdq-analysis/pkg/componentdetectionquery.go

```diff
@@ -38,7 +38,7 @@ type K8sInfoClient struct {
 
 // CDQ analyzer
 // return values are for testing purpose
-func CloneAndAnalyze(k K8sInfoClient, gitToken, namespace, name, context, devfilePath, dockerfilePath, URL, Revision, DevfileRegistryURL string, isDevfilePresent, isDockerfilePresent bool) (map[string][]byte, map[string]string, map[string]string, map[string][]int, error) {
+func CloneAndAnalyze(k K8sInfoClient, gitToken, namespace, name, context, devfilePath, dockerfilePath, URL, Revision, DevfileRegistryURL string, isDevfilePresent, isDockerfilePresent bool) (map[string][]byte, map[string]string, map[string]string, map[string][]int, string, error) {
```

Comment:
This is a breaking API change.  Do we need to notify any consumers?

Reply:
We’re the only consumer of this function and go module, no?